### PR TITLE
Number guessing game

### PR DESCRIPTION
**Pull request by adedayoprcs**
I created a simple number guessing game.

### CODE REVIEW

File: Number-Guessing/number-guessing
Comment (iamwatchdogs) [inline Number-Guessing/number-guessing:1]: Please rename the file with the extension as `.py`

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Number-Guessing/number-guessing
Comment (adedayoprcs) [reply Number-Guessing/number-guessing:1]: File has been renamed.

Comment (iamwatchdogs) [reply Number-Guessing/number-guessing:1]: @adedayoprcs Please check `Number-Guessing/number-guessing` file within your PR. Your filename still doesn't contain the `.py` file extension.

**Review by iamwatchdogs — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Pull request closed without merging**